packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -179,12 +193,11 @@ type ChangesetResult =
   | {
       errors: ChangesetError[];
       status: 'unprocessed';
-      updatedPurchase: UpdatedPurchase | null;
```

Comment:
Is the key `updatedPurchase` completely missing from the response or `null`?

Reply:
It would be entirely missing - you shouldn't ever get `unprocessed` unless there was an error, in which case that should be the only thing in your payload - we can't give you a calculated order if there was a failure.